x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -129,6 +129,11 @@ public Query existsQuery(QueryShardContext context) {
             return new DocValuesFieldExistsQuery(name());
         }
 
+        @Override
+        public boolean isAggregatable() {
```

Comment:
This approach makes sense to me. I wondered if we should `FieldMapper#isAggregatable` more robust/ general, now that there are field types that produce field data but don't support aggregations. But I think `dense_vector` is the only example right now, so we can avoid generalizing too early.